python/dolfinx/fem/function.py

```diff
@@ -9,8 +9,7 @@
 
 import typing
 
-if typing.TYPE_CHECKING:
-    from dolfinx.mesh import Mesh
+from dolfinx.mesh import Mesh
```

Comment:
Docs don't build with this if statement

Reply:
See https://github.com/FEniCS/dolfinx/pull/1957.